src/python/utils/stopword_model.py

```diff
@@ -0,0 +1,33 @@
+from sets import Set
+from datetime import datetime
+
+
+class StopWordModel(object):
+
+    # Top Level Fields
+    PKEY = 'pKey'
+    SORTKEY = 'sortKey'
+    VERSION = 'version'
+
+    # Assign Top Level Fields
+    VALID_TOP_LEVEL_FIELDS = [PKEY,
+                              SORTKEY,VERSION]
+
+    def __init__(self):
+        self._top_level_fields = Set(self.VALID_TOP_LEVEL_FIELDS)
+        self._info = {self.PKEY:'',self.SORTKEY:-1,self.VERSION:-1}
+
+    def set_data(self, pKey, sortKey,version):
```

Comment:
Can you split this into with_pkey(), with_sort_key(), and with_version()? This just makes it a bit more readable when a StopWordModel object is instantiated since it each function describes which field it's setting.